README.md

```diff
@@ -76,20 +76,20 @@ yarn test
 ## Overview
 Following is a conceptual explanation of the primitives.
 
-### BBS+ Signature
-BBS+ signature allow for signing an ordered list of messages, producing a signature of constant size independent of the number
+### BBS Signature
+BBS signature allow for signing an ordered list of messages, producing a signature of constant size independent of the number
```

Comment:
Please add a short disclaimer here about BBS/BBS+, similar to what is written in the [IRTF draft](https://identity.foundation/bbs-signature/draft-irtf-cfrg-bbs-signatures.html), but shorter, and mentioning BBS and BBS+:

> The name BBS is derived from the authors of the original academic work by Dan Boneh, Xavier Boyen, and Hovav Shacham [[BBS04](https://identity.foundation/bbs-signature/draft-irtf-cfrg-bbs-signatures.html#BBS04)], where the scheme was first described as part of a group signatures protocol. Soon after, the scheme was described by Camenisch and Lysyanskaya as a stand-alone signatures scheme in [[CL04](https://identity.foundation/bbs-signature/draft-irtf-cfrg-bbs-signatures.html#CL04)], for anonymous credentials applications. Later, Au, Susilo an Mu presented the first, provably secure version of BBS Signatures in [[ASM06](https://identity.foundation/bbs-signature/draft-irtf-cfrg-bbs-signatures.html#ASM06)]. Following, works by Camenisch, Drijvers and Lehmann [[CDL16](https://identity.foundation/bbs-signature/draft-irtf-cfrg-bbs-signatures.html#CDL16)] and by Barki, Brunet, Desmoulins and Traore [[BBDT16](https://identity.foundation/bbs-signature/draft-irtf-cfrg-bbs-signatures.html#BBDT16)], proved the security of the scheme in settings where more efficient computations are possible, thereby improving performance. Finally, in 2023, Tessaro and Zhu, presented in [[TZ23](https://identity.foundation/bbs-signature/draft-irtf-cfrg-bbs-signatures.html#TZ23)] further performance improvements, shrinking the BBS signature. This document is mainly based on that work.

Reply:
I added a simple disclaimer at the top. Do you think there's something more to add there?